Makefile

```diff
@@ -127,11 +165,9 @@ bazel-generate:
 # find or download controller-gen
 # download controller-gen if necessary
 controller-gen:
-ifeq (, $(shell which controller-gen))
+ifeq (, $(CONTROLLER_GEN))
 	@{ \
 	go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.10.0 ;\
 	}
-CONTROLLER_GEN=$(GOBIN)/controller-gen
-else
-CONTROLLER_GEN=$(shell which controller-gen)
+CONTROLLER_GEN = $(shell command -v controller-gen)
```

Comment:
Could you please elaborate on why you chose to switch from `which` to `command -v`? While the commands are not exactly the same it shouldn't cause trouble in this context. It just may seem inconsistent to causal reader.

Reply:
I thought it would minimize dependencies as `command` is part of the `bash` package and the `which` command has his own `which` package. Of course I am assuming everybody is using `bash` and maybe it's a big assumption. I think this change is not important so I can revert it if you want.

Reply:
What I really meant is that on some places (e.g. `CONTAINER_CMD`) there is `which` and on other places (e.g. `CONTROLLER_GEN`) there is `command -v`. I don't mind using either of those, but we should try to make it consistent unless there are some reasons why not to.

Reply:
Yeah, you are right, I realized yesterday when I was going to revert that change.  Other shells like `zsh` also have the built-in command `command -v`  so I will keep this change and make it consistent within the Makefile